django_munin/munin/migrations/0002_test.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 1.9.8 on 2016-12-05 01:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ("munin", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
Je viens de tester la migration, et en effet, en l'état actuel de la PR, elle ne passe pas parce que la table existe déjà. En ajoutant l'opération `RunSQL` suivante, ça fonctionne. Tu peux l'ajouter, stp ?
```suggestion
        migrations.RunSQL("DROP TABLE IF EXISTS munin_test"),
        migrations.CreateModel(
```

Reply:
J'ai fait la modif